src/brogue/IO.c

```diff
@@ -1489,6 +1489,25 @@ void getCellAppearance(short x, short y, enum displayGlyph *returnChar, color *r
     if (rogue.trueColorMode
         && playerCanSeeOrSense(x, y)) {
 
+        if (displayDetail[x][y] == DV_DARK) {
+            applyColorMultiplier(&cellForeColor, &inDarknessMultiplierColor);
+            applyColorMultiplier(&cellBackColor, &inDarknessMultiplierColor);
+
+            applyColorAugment(&cellForeColor, &purple, 10);
+            applyColorAugment(&cellBackColor, &white, -10);
+            applyColorAverage(&cellBackColor, &purple, 20);
+        } else if (displayDetail[x][y] == DV_LIT) {
+
+            colorMultiplierFromDungeonLight(x, y, &lightMultiplierColor);
+            normColor(&lightMultiplierColor, 175, 50);
+            //applyColorMultiplier(&cellForeColor, &lightMultiplierColor);
+            //applyColorMultiplier(&cellBackColor, &lightMultiplierColor);
+            applyColorAugment(&cellForeColor, &lightMultiplierColor, 5);
+            applyColorAugment(&cellBackColor, &lightMultiplierColor, 5);
+        }
+    } else if (rogue.displayStealthRangeMode
```

Comment:
These two conditional bodies are currently identical, but I've broken them out in anticipation of changing how the 2nd case is drawn while not affecting the first.

Reply:
Will that be in this PR? Or do you have this change lined up for later?

Reply:
Apologies, time got away from me.  I haven't yet made the initial attempt to use the mentioned dark blue.  But I should get a chance in the next 6 hours or so.

I'm also completely happy if you'd rather merge this, and I make that attempt in a new one (or would you rather not have the blocks unfolded until the color customization actually appears?)

Reply:
No it's cool, I didn't imagine there was going to be a variation in colour - do you think the blue will need tweaking or one should stay purple?

Reply:
I suppose I was implicitly carrying the assumption that the simpler color mode would stay purple, but that doesn't help with the question of caustic gas.  Ok, re-folded back together.

I've also switched out the purple for the "player in darkness" color, which is a bit bluer.  The "in darkness multiplier", which darkens things in shadow, also became more intense.